src/modules/appointment/models/appointment.entity.ts

```diff
@@ -59,6 +61,13 @@ export class AppointmentEntity {
   })
   status: string;
 
+  // @Column({
+  //   name: 'cost_amount',
+  //   type: 'decimal',
+  //   nullable: true,
```

Comment:
why did you comment this?

Reply:
@imadiaz That was already there, but I think the old changes I had were mixed with the new ones from the develp branch